test/testcafe/spec/ChallengeCustomAppPush_spec.js

```diff
@@ -76,7 +84,9 @@ test.requestHooks(mockCustomAppSendPush)(
   async t => {
     const challengeCustomAppPushPageObject = await setup(t);
     await checkA11y(t);
-    await t.expect(await challengeCustomAppPushPageObject.isCustomAppSendPushForm()).ok();
+    if (!userVariables.v3) {
+      await t.expect(await challengeCustomAppPushPageObject.isCustomAppSendPushForm()).ok();
+    }
```

Comment:
For each condition like this in this test, can you add a brief comment above as to why we are only running it against v2 (if negative or v3 if positive)? Just so we can understand what is being tested and what gaps there are.